ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,1136 @@
+
+<https://semopenalex.org/property/hasBestOaLocation> a owl:ObjectProperty;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "hasBestOaLocation" .
+
+<https://semopenalex.org/property/sourceType> a owl:DatatypeProperty;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "sourceType" .
+
+<https://semopenalex.org/property/apcUsd> a owl:DatatypeProperty;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "apcUsd" .
+
+<https://semopenalex.org/property/fatcatId> a owl:DatatypeProperty;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "fatcatId" .
+
+<https://semopenalex.org/property/abbreviatedName> a owl:DatatypeProperty;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "abbreviatedName" .
+
+<https://semopenalex.org/property/hasHostOrganization> a owl:ObjectProperty;
+  rdfs:isDefinedBy <https://semopenalex.org/vocab#>;
+  rdfs:label "hasHostOrganization" .
+
+<https://semopenalex.org/class/PublisherShape> a sh:NodeShape;
+  sh:property [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/citedByCount>;
+      sh:datatype xsd:integer
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/countsByYear>;
+      sh:class <https://semopenalex.org/class/CountsByYear>
+    ], [ a sh:PropertyShape;
+      sh:path <http://www.geonames.org/ontology#countryCode>;
+      sh:datatype xsd:string
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/hasParentPublisher>;
+      sh:class <https://semopenalex.org/class/Publisher>
+    ], [ a sh:PropertyShape;
+      sh:path <http://purl.org/dc/terms/created>;
+      sh:datatype xsd:date
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/level>;
+      sh:datatype xsd:integer
+    ], [ a sh:PropertyShape;
+      sh:path foaf:name;
+      sh:datatype xsd:string
+    ], [ a sh:PropertyShape;
+      sh:path <http://purl.org/spar/bido/h-index>;
+      sh:datatype xsd:integer
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/i10Index>;
+      sh:datatype xsd:integer
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/ror>;
+      sh:datatype xsd:string
+    ], [ a sh:PropertyShape;
+      sh:path <http://purl.org/dc/terms/modified>;
+      sh:datatype xsd:date
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/2YrMeanCitedness>;
+      sh:datatype xsd:float
+    ], [ a sh:PropertyShape;
+      sh:path <https://dbpedia.org/ontology/alternativeName>;
+      sh:datatype xsd:string
+    ], [ a sh:PropertyShape;
+      sh:path <https://semopenalex.org/property/worksCount>;
+      sh:datatype xsd:integer
+    ];
+  sh:targetClass <https://semopenalex.org/class/Publisher> .
+
+<https://semopenalex.org/property/hasVenue> rdfs:domain <https://semopenalex.org/class/Location>;
+  rdfs:range <https://semopenalex.org/class/Source> .
+
+[] rdf:first [
```

Comment:
what is this supposed to be? this is an rdf-list identified by a blank node? where is it used